### PR TITLE
TECH-1252 Feature: add README documentation

### PR DESCRIPTION
**Pull request by multinauta**
Refer: https://app.clickup.com/t/3077149/TECH-1252

Why I'm adding these lines of code?

- to document installation and use

### CODE REVIEW

**Comment by hiro-riveros** [comment]
Task linked: [TECH-1252 Documentar Shipit-Wordpress](https://app.clickup.com/t/3077149/TECH-1252)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# SHIPIT-WORDPRESS
+## Desription
+
+The following project is a plugin for wordpress which interacts with the woocommerce plugin to be able to generate pricing, import sales and update order statuses. To testing in local we must install a wordpress instance.
```
Comment (Zeaneth) [inline README.md:4]:
```suggestion
The following project is a plugin for WordPress that interacts with the WooCommerce plugin to generate pricing, import sales, and update order statuses. To test in local, we must install a WordPress instance.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+## Requeriments
+
+
+| Technology | Versión |
```
Comment (Zeaneth) [inline README.md:9]:
```suggestion
| Technology | Version |
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+| ------ | ------ |
+| Apache | Last |
+| Mysql/Mariadb | Last |
+| PHP| Last |
+| Wordpress| Last |
+| Woocommerce | Last |
```
Comment (Zeaneth) [inline README.md:15]:
```suggestion
| WooCommerce | Last |
```

**Review by Zeaneth — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (multinauta) [reply README.md:4]: Thanks!

File: README.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+## Install Woocommerce
+- Go to: Plugins > Add New.
+- Search for “WooCommerce”.
+- Click Install Now.
+- Click Activate Now and you’re ready for the WooCommerce Wizard.
+
+## Install Shipit in Wordpress
+[How install shipit](https://shipitcl.zendesk.com/hc/es-419/articles/360016135074--C%C3%B3mo-integrar-mi-tienda-de-WooCommerce-con-Shipit-)
+
+## Others products necesary
```
Comment (Zeaneth) [inline README.md:115]:
```suggestion
## Others necessary products
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
+# SHIPIT-WORDPRESS
+## Desription
+
+The following project is a plugin for WordPress that interacts with the WooCommerce plugin to generate pricing, import sales, and update order statuses. To test in local, we must install a WordPress instance.
+
+## Requeriments
```
Comment (Zeaneth) [inline README.md:6]:
```suggestion
## Requirements
```

**Review by Zeaneth — COMMENTED** [review]
I hadn't seen the other two 😅

File: README.md (hunk 4 above)
Comment (multinauta) [reply README.md]: Done!

**Review by Zeaneth — APPROVED** [review]

**Review by luispascuali — APPROVED** [review]

**Review by gcamposm — APPROVED** [review]

**Review by David25848 — APPROVED** [review]

**Review by Barbycris — APPROVED** [review]

**Review by mgraterol — APPROVED** [review]

**Review by HaNdlezz — APPROVED** [review]

**Pull request merged**